packages/server/src/tests/test-couchdb.ini

```diff
@@ -0,0 +1,2 @@
+[log]
```

Comment:
I've taken the opportunity here to calm the CouchDB logs down. By default they log every single HTTP request, which is an unbelievable amount of logging. This made `DEBUG=testcontainers*` unusable, whereas with `level = warn` the output is a little less huge and more useful.